Tasks/NodeToolV0/nodetool.ts

```diff
@@ -125,7 +126,7 @@ async function queryLatestMatch(versionSpec: string, installedArch: string): Pro
     }
 
     let versions: string[] = [];
-    let dataUrl = "https://nodejs.org/dist/index.json";
+    let dataUrl = nodejsMirror + "/index.json";
```

Comment:
If provided URL contains slash (/) at the end it will not work. Could you please trim slashes at the end before using it?